bookmark/bookmark.js

```diff
@@ -0,0 +1,16 @@
+javascript: (function () {
+  const { hostname } = window.location;
+
+  const protocol = window.location.protocol === 'http:' ? 'ws' : 'wss';
+  const getAppId = (location) => {
+    const prefix = '/sense/app/';
+    const start = location.pathname.indexOf(prefix) + prefix.length;
+    const end = location.pathname.indexOf('/', start);
+    const appId = location.pathname.substring(start, end !== -1 ? end : location.pathname.length);
+    return appId;
+  };
+  const appId = getAppId(window.location);
+  const engineUrl = `${protocol}://${hostname}:9076/app/${appId}`;
```

Comment:
We need to get the port from the URL as well.

Tried it now with Sense Desktop but that does not work since it runs on `4848`

Reply:
It works for me, with Sense Desktop. Shouldn't it run on port 9076?